atg_canvas_course_enable_app.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Parse command line arguments
+while getopts "f:h:k:t:" opt; do
+    case $opt in
+        f)
+            CSV_FILE="$OPTARG"
+            ;;
+        h)
+            URL="$OPTARG"
+            ;;
+        k)
+            API_KEY="$OPTARG"
+            ;;
+        t)
+            TOOL_ID="$OPTARG"
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            exit 1
+            ;;
+    esac
+done
+
+# Check if CSV_FILE, URL, API_KEY, and TOOL_ID are set
+if [ -z "$CSV_FILE" ] || [ -z "$URL" ] || [ -z "$API_KEY" ] || [ -z "$TOOL_ID" ]; then
+    echo "Usage: $0 -f <csv_file> -h <url> -k <api_key> -t <tool_id>"
+    exit 1
+fi
+
+headers=1
+
+
+# Skip the headers if the headers variable is set to 1
```

Comment:
Why would you want to skip vs. keep the headers? It looks like it's either using or discarding the headers from the CSV file input, so does it depend on how the CSV file is set up?

Reply:
Yea I was thinking if the CSV didn't have headers there should be a fall back. Depending on how flexible the script can be we can ether add a header variable/input or just assume headers will always be in the csv file.